configs/templates/jedi-ufs-all/spack.yaml

```diff
@@ -23,14 +23,17 @@ spack:
    - jedi-tools-env
    - jedi-ufs-env
    - jedi-um-env
-   - global-workflow-env
    # Turn off Python variant for now, because this
    # leads to duplicate packages being built (all
-   # of them are covered in jedi-base-env)
+   # of them are covered in jedi-base-env,
```

Comment:
The changes in the workflows above seem to suggest that `jedi-ufs-all` is getting deprecated. Is that correct? If so, should this template be removed?

Reply:
I was thinking the same, especially since there are so many templates and one of them (`hpc-dev-v1`) is quite similar to `jedi-ufs-all`. Let me remove it.

Reply:
Actually, we can't. The "concretize only" CI tests still use it (`./.github/workflows/concretize.yaml`). I don't want to remove them now, but I'll create an issue to clean up the templates and in particular consider removing `jedi-ufs-all`.

Reply:
Sounds good! Thanks!

Reply:
See https://github.com/NOAA-EMC/spack-stack/issues/341

Reply:
A post-release feature!